Dockerfile

```
@@ -38,5 +27,3 @@ RUN        pip install --user ./
 
```

Based on the current status of the application, it is also needed to install the latest master for `topic-db`.

```suggestion
RUN        pip uninstall topic-db -y
RUN        git clone https://github.com/brettkromkamp/topic-db.git
RUN        pip install --user -r ./topic-db/requirements.txt
RUN        pip install --user ./topic-db
```